packages/ui/vue-ui/src/components/CrossmintPaymentElement.vue

```diff
@@ -76,12 +78,13 @@ onUnmounted(() => {
 });
 
 watch(
-    () => [props.recipient, props.mintConfig, props.locale, props.whPassThroughArgs],
+    () => [props.recipient, props.mintConfig, props.locale, props.currency, props.whPassThroughArgs],
```

Comment:
have to make the similar change in CrossmintFiatPaymentElement_OLD

Reply:
might as well add it to CryptoEmbeddedCheckoutIFrame as well?

Reply:
Isn't this accomplished by having currency and locale in `packages/core/base/src/types/embed/index.ts` ?

Reply:
nah, check the useEffect here for example https://github.com/Crossmint/crossmint-client-sdk/blob/deb98cae454a6c94e5a78bc83a85f565b842fba3/packages/ui/react-ui/src/components/embed/fiat/FiatPaymentElement_OLD.tsx#L49